python/desc/rubin_roman_sims/visit_handling.py

```diff
@@ -1,25 +1,21 @@
 import os
 import glob
 import json
-from collections import defaultdict
+from itertools import cycle
 
 
 __all__ = ['make_visit_tranches', 'omit_existing_ccd_visits']
 
 
 def omit_existing_ccd_visits(ccd_lists, output_dir):
-    trimmed_ccd_lists = {}
-    for visit, ccd_candidates in ccd_lists.items():
-        ccds = []
-        for det_name in ccd_candidates:
-            pattern = os.path.join(output_dir,
-                                   f'amp_*{visit}*{det_name}*.fits*')
-            amp_files = glob.glob(pattern)
-            if not amp_files:
-                ccds.append(det_name)
-        trimmed_ccd_lists[visit] = ccds
-    return trimmed_ccd_lists
-
+    return {
+        visit: [
+            det_name
+            for det_name in ccd_candidates
+            if not glob.glob(os.path.join(output_dir, "amp_*.fits*"))
```

Comment:
I think the glob pattern still needs to be formatted with the visit and det_name, i.e.,
```
            if not glob.glob(os.path.join(output_dir, f"amp_*{visit}*{det_name}*.fits*"))
```
There's some test code in the `tests` folder that exercises this.

Reply:
Should be corrected now